client/src/plugins/camunda-plugin/deployment-tool/__tests__/DeploymentToolSpec.js

```diff
@@ -498,6 +498,25 @@ describe('<DeploymentTool>', () => {
     });
 
 
+    it('should fetch the executionPlatformVersion but no version is available', async () => {
+
+      // given
+      const deploySpy = sinon.spy();
+      const getVersionSpy = sinon.spy(() => { return { version: null }; });
+      const activeTab = createTab({ name: 'foo.bpmn' });
+      const {
+        instance
+      } = createDeploymentTool({ activeTab, deploySpy, getVersionSpy });
+
+      // when
+      await instance.deploy();
+
+      // then
+      expect(getVersionSpy).to.have.been.calledOnce;
+      expect(getVersionSpy).to.not.have.been.called;
```

Comment:
I expect this to at throw an error, but it doesn't. I'm not sure if this test is ran.

Reply:
Never mind, it was `npm run client:test` that was needed to run the tests.